integrating-melodi-analytics/react-component-new.mdx

```diff
@@ -0,0 +1,173 @@
+---
+title: 'React Component'
+description: 'How to use the Melodi React component'
+---
+
+# melodi-sdk-react
+
+A small sdk for adding Melodi's feedback react component to your app. The feedback widget lets you add thumbs up/thumbs down buttons anywhere in your app and a popover to record feedback from your users about your llm generated text. You can view responses in the Feedback Inbox in your Melodi account.
+
+<Note>This library has `react` and `react-dom` (both v18.2 or higher) as peer dependencies.</Note>
+
+## Getting started
+
+### To add the melodi widget to your react app, follow these simple steps:
+
+1. Install the library
+   `npm install @melodi/melodi-sdk-react`
+
+2. Get a publishable API key from your Melodi Account.
+
+   - Log into your Melodi account and go to [settings](https://app.melodi.fyi/admin/settings).
+   - Click "Create publishable api key"
+   - Copy the newly created key
+
+3. Add the MelodiAuthProvider with your api key
+
+```jsx
+<MelodiAuthProvider apiKey="YOUR_API_KEY_HERE">
+  {children}
+</MelodiAuthProvider>
+```
+
+4. Add the MelodiFeedbackWidget and pass along the relevant info
+
+```jsx
+<MelodiFeedbackWidget
+  thread={{
```

Comment:
This will work and is fine for the first pass, but I think what we ultimately want to get people to do is send us the thread via the typescript sdk and then reference the threadId or externalThreadId in the feedback widget.

The upside of that approach is that we will have all of the threads, including ones that never receive feedback.

The downside is that it involves both the typescript sdk and the react widget and it's ever so slightly more complicated to get up and running.

Happy to have a discussion about it, just thinking out loud here

Reply:
I like it! I added tabbed views so now you can see both